docs/governance/politeia.md

```diff
@@ -2,54 +2,56 @@
 
-Politeia is built around the concept of _transparent censorship_, using [dcrtime](https://github.com/decred/dcrtime). Users cannot be silently censored, they can prove that censorship has occurred. When a proposal/comment is submitted, a censorship token is generated. Content creators can use these tokens to demonstrate that their specific proposal/comment was submitted, in a particular form, at a particular point in time.
+Politeia is built around the concept of **transparent censorship**, using [dcrtime](https://github.com/decred/dcrtime). Users cannot be silently censored; they can prove that censorship has occurred. When a proposal/comment is submitted, a censorship token is generated. Content creators can use these tokens to demonstrate that their specific proposal/comment was submitted, in a particular form, at a particular point in time.
 
-**Politeia users have cryptographic identities** (pub/priv key pairs) that can be used to demonstrate censorship of one's submissions. When you register for a Politeia account, a key-pair identity will be generated for you, this is stored in your browser by default.
+Censorship tokens are tied to a **cryptographic identity** (pub/priv key pair) created when you register for a Politeia account. This identity is only used for demonstrating that your submissions have been censored. This key-pair is stored in your browser by default. For now, it is important to **verify your account from the same browser you used to sign up**. 
 
-If you change browser/device you can import your Pi identity key or generate a new one, but for now it is important to *verify your account from the same browser you used to sign up*. The cryptographic Pi identity is only used for demonstrating that your submissions have been censored.
+### Proposal Lifecycle
 
-**Submitting and approving proposals**
+Below is the basic process for submitting and evaluating proposals:
 
-Politeia's aim is to serve as the decision-making force behind the Decred Decentralized Autonomous Entity (DAE). This is an ambitious aim, Politeia and its accompanying processes are in an experimental stage and thus subject to change.
+1. A proposal is submitted.
+2. The proposal is reviewed by Politeia admins, spam is censored. 
+3. Valid proposals appear publicly on Politeia. They are open for discussion, but voting does not begin immediately, and the proposal can be edited by its owner in response to community feedback. 
+4. The proposal owner authorizes voting to start. An admin then triggers the start of voting.	
+5. The ticket-voting interval of 2016 blocks (~1 week) begins. A snapshot of the live ticket pool is taken at 256 blocks prior to the start of voting. Every ticket in this snapshot can vote 'Yes' or 'No' on the proposal. Tickets bought after the snapshot for a proposal cannot vote on it. Tickets that vote on-chain can still be used to vote on a proposal if they were live when the snapshot was taken.
```

Comment:
Might be useful to include:

Politeia votes are not on-chain votes.  They are stored off-chain.

Reply:
The wording here is tricky. I think it's best not to get too into the details of on-chain vs off-chain voting. Have taken a stab at clearer text. Should definitely be checked for technical accuracy.

>The ticket-voting interval of 2016 blocks (~1 week) begins. A snapshot of the live ticket pool is taken at 256 blocks prior to the start of voting. Every ticket in the pool when this snapshot was taken can vote 'Yes' or 'No' on the proposal. Tickets bought after the snapshot cannot vote on the proposal. If a ticket is called to vote on-chain during the ticket-voting interval (to validate blocks or vote on consensus rule changes), it still has until the end of the ticket-voting interval to vote on the proposal.